bibliopixel/drivers/SPI/driver.py

```diff
@@ -0,0 +1,42 @@
+from .. channel_order import ChannelOrder
+from .. driver_base import DriverBase
+from ... util.enum import resolve_enum
+from . import interfaces
+from enum import IntEnum
+# SubDriver imports
+from . APA102 import APA102
+from . LPD8806 import LPD8806
+from . WS281X import WS281XSPI
+from . WS2801 import WS2801
+
+
+class SPI_LEDTYPE(IntEnum):
+    APA102 = 0
+    DOTSTAR = 0
+    WS2801 = 1
+    WS281X = 2
+    WS2812 = 2
+    WS2812B = 2
+    NEOPIXEL = 2
+    LPD8806 = 3
+
+
+SPI_DRIVERS = [
+    APA102,
+    WS2801,
+    WS281XSPI,
+    LPD8806
+]
+
+
+class SPI(DriverBase):
```

Comment:
Yeah... I thought about doing it that way, but the intent here was to keep this class a DriverBase like everything else so that the system would see it the same as everything else. I realize that doing the way you suggest would look syntactically the same in code but my concern is that as we bolt features like PixelWeb back in and start using a lot of reflection to determine what is what, SPI will get ignored because it's not an actual Driver and then we'll have to do special handling. So, yes, more complex now, but simpler later.